src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
+	register "sata_ports_enable[1]" = "1" # M.2 2280
+	register "sata_ports_dev_slp[0]" = "1"
+	register "sata_ports_dev_slp[1]" = "1"
+
+	device domain 0 on
+		device ref igpu on
+			subsystemid 0x8086 0x2212
+			register "ddi_portA_config" = "1" # HDMI on port A
+			register "ddi_portB_config" = "1" # DP on port B
+			register "ddi_ports_config" = "{
+				[DDI_PORT_A] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_B] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_1] = DDI_ENABLE_HPD,
+				[DDI_PORT_2] = DDI_ENABLE_HPD,
+			}"
+		end
+
+		# SATA
+		device ref pcie_rp11 off end
+		device ref pcie_rp12 off end
```

Comment:
RP12 is not off, it is WIFI... Please update accordingly.

Reply:
WiFi uses CLKSRC 4

Reply:
CNVi device missing in the devicetree here.

Reply:
[Added](https://github.com/Dasharo/coreboot/pull/576/files#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2R77) CNVi device. Not sure how to set the `wake` register. Found it set to `GPE0_PME_B0` in most platforms, searched through the mtl-h schematics for reference, but found no such thing, only `CPU_PCIE_WAKE` on `GPP_V14`. Similar for the VP2430 schematics, only found a `PM_N_WAKE` signal connected to a `wake#` pin

Reply:
[Added](https://github.com/Dasharo/coreboot/pull/576/files#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2R56) on RP12. Got confused since it's not labeled like all the other root ports for some reason, and the pins are named as SATA

Reply:
RP11 is SATA, RP12 is connected to KEY_E_WIFI slot. You are confusing things or looking at different schematics than I do.